lessons/Data-Science/contracts-tables-visualizations/langs/en-us/pages/functions-for-tables-cont.adoc

```diff
@@ -28,7 +28,7 @@ Type @show{(code '(count animals-table "legs"))} into the Interactions Area and
 
 @fitbruby{10em}{@ifsoln{count}}{function-name} (@fitbruby{20em}{@ifsoln{animals-table}}{table-name {two-colons} Table}, @fitbruby{15em}{@ifsoln{Answers will vary!}}{column-name {two-colons} String})
 
-@n Tables that summarize data with a count are commonly used in the real world. Give two examples of where you've seen them before:
+@n These tables are called @vocab{Frequency Tables}, which are used often in the real world. Give two examples of where you've seen them before:
```

Comment:
This doesn't read coherently. I'd suggest

> These tables are called @vocab{Frequency Tables}. Give two examples of where you've seen them used to summarize data outside of math class:

Reply:
love it